test/testutil/storage/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+add_library(base_fs_test
+  base_fs_test.hpp
```

Comment:
please set in clion in code style for cmake tab size = 4, Indent =2, Continuation indent = 4. And reformat all your cmake code with cmd+option+L

Reply:
done